includes/Main.php

```diff
@@ -285,10 +285,18 @@ public function get_documents_for_email( $email_id, $order ) {
 		$attach_documents = array();
 
 		foreach ( $documents as $document ) {
-			// Pro not activated, only attach Invoice
-			if ( ! function_exists( 'WPO_WCPDF_Pro' ) && 'invoice' !== $document->get_type() ) {
+			/**
+			 * Filter documents for email attachment based on plugin availability.
+			 * 
+			 * - If Pro plugin is not active, only allow 'invoice' and 'proposal' document types
+			 * - 'proposal' document type is only available if the Order Proposal plugin is active
+			 * - Skip documents that don't meet these criteria
+			 */
+			if ( 
+				( ! function_exists( 'WPO_WCPDF_Pro' ) && ! in_array( $document->get_type(), array( 'invoice', 'proposal' ) ) ) ||
```

Comment:
This doesn't seem correct to me. The Proposal is not part of the `WPO_WCPDF_Pro`.